tests/devices/i19/test_unit_hutch_conditional_shutter.py

```diff
@@ -0,0 +1,39 @@
+from unittest import TestCase
+
+import pytest
+
+from dodal.devices.i19.shutter import (
+    HutchConditionalShutter,
+    HutchInvalidError,
+    HutchState,
+)
+
+
+async def unit_test_shutter_created_without_raising_errors(
+    hutch_controlling_shutter: HutchState,
+):
+    fake_prefix = "I74"
+    shutter_name = "GrandOldShutter"
+    hcs = HutchConditionalShutter(fake_prefix, hutch_controlling_shutter, shutter_name)
+    test_case = TestCase()
+    test_case.assertIsInstance(hcs, HutchConditionalShutter)
+
+
+async def test_shutter_created_for_eh_one_without_raising_errors():
+    hutch_from_which_shutter_is_controlled = HutchState.EH1
+    await unit_test_shutter_created_without_raising_errors(
+        hutch_from_which_shutter_is_controlled
+    )
+
+
+async def test_shutter_created_for_eh_two_without_raising_errors():
+    hutch_from_which_shutter_is_controlled = HutchState.EH2
+    await unit_test_shutter_created_without_raising_errors(
+        hutch_from_which_shutter_is_controlled
+    )
```

Comment:
Could: You can use `@pytest.mark.parametrize` to do this for you:
```suggestion
@pytest.mark.parametrize(
    "hutch_from_which_shutter_is_controlled", [HutchState.EH1, HutchState.EH2]
)
async def test_shutter_created_without_raising_errors(
    hutch_from_which_shutter_is_controlled,
):
    await unit_test_shutter_created_without_raising_errors(
        hutch_from_which_shutter_is_controlled
    )
```
This will create two tests called `test_shutter_created_without_raising_errors[EH1]` and `test_shutter_created_without_raising_errors[EH2]`. 

In the case of just having the two options it's neither here nor there but when there's like 3 or 4 it tidies up a lot of the copy/paste.